mvc/app/view/home/index.php

```diff
@@ -0,0 +1 @@
+Hello <?php echo $data['name'];?>
```

Comment:
what is this? I think this is for testing purpose.

Reply:
Yes it is for testing purpose.